.github/workflows/cut-release.yml

```diff
@@ -0,0 +1,59 @@
+name: "cut-release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'version'
+        required: false
+jobs:
+  cut_release:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code 
+        uses: actions/checkout@v3
+      - name: "Build & test"
+        run: |
+          sudo apt-get install build-essential -y
+          ./autogen.sh
+          ./configure
+          make distcheck
+      - name: Update VERSION file on checkout 
+        run: |
+          set -x
+          sudo apt-get install -y python3-semver
+          if [ -z "$NEW_VERSION" ]; then
+             echo "Current Version: $(cat VERSION)"
+             pysemver bump patch $(cat VERSION) > VERSION
+          else
+             echo $NEW_VERSION > VERSION
+          fi
+          export NEW_VERSION=$(cat VERSION)
+          echo "::set-output name=NEW_VERSION::$NEW_VERSION" 
+        id: bump-version-file  
+        env:
+          NEW_VERSION: ${{ inputs.version }}
+      - name: Commit VERSION file 
+        uses: EndBug/add-and-commit@v9
+        with:
+          add: 'VERSION'
+          message: bump semantic version
+          committer_name: GitHub Actions
+          committer_email: actions@github.com
+      - name: Create Tag
+        uses: rickstaa/action-create-tag@v1
+        with:
+          tag: "v${{ steps.bump-version-file.outputs.NEW_VERSION }}"
+          message: "v{{ steps.bump-version-file.outputs.NEW_VERSION }} release"
```

Comment:
Let's use a tag description of `Release numactl {{ steps.bump-version-file.outputs.NEW_VERSION }}` here.

See https://github.com/numactl/numactl/releases/tag/v2.0.14 for an example.